config.yaml

```
@@ -1,45 +1,50 @@
 options:
   path:
-      type: string
-      default: ""
-      description: >
-        The path of the GitHub repository to register the self-hosted runners. Either an
-        organization name or repository in 'org/repo' format.
+    type: string
+    default: ""
+    description: >
+      The path of the GitHub repository to register the self-hosted runners. Either an
+      organization name or repository in 'org/repo' format.
+  group:
+    type: string
+    default: "default"
+    description: >
+      The organization runner group to register the self-hosted runner under. This has no
+      effect on runners under a repository.
   token:
-      type: string
-      default: ""
-      description:
-        The GitHub Personal Access Token for registering the self-hosted runners.
+    type: string
+    default: ""
+    description: The GitHub Personal Access Token for registering the self-hosted runners.
   virtual-machines:
-      type: int
-      default: 0
-      description: >
-        The number of virtual machine runners. This charm will spawn or destroy virtual
-        machines runners to match this setting.
+    type: int
+    default: 0
+    description: >
+      The number of virtual machine runners. This charm will spawn or destroy virtual
+      machines runners to match this setting.
   vm-cpu:
-      type: int
-      default: 2
-      description: >
-        The number of CPUs used per virtual machine runner.
+    type: int
+    default: 2
+    description: >
+      The number of CPUs used per virtual machine runner.
   vm-memory:
     type: string
     default: 7GiB
     description: >
-        The memory size used per virtual machine runner.
+      The memory size used per virtual machine runner.
```

Would this phrasing help be clear about the accepted units? I think sometimes users might put just a number of different units as the input :)
```suggestion
      Amount of memory to allocate per virtual machine runner. Positive integers with GiB suffix.
```
Taken from [multipass documentation](https://multipass.run/docs/launch-command) :D